### DISCUSSION TITLE
绳匠们请看好自己的邮箱

### DISCUSSION THREAD

**Discussion by Buer-Nahida** [share121/inter-knot#715] [其他]
![图片](https://github.com/user-attachments/assets/4b19fab7-244e-407b-8707-a9d79be03923)

> [!CAUTION]
>
> 绳匠们回复完帖子后记得点下`Unsubscribe`按钮，否则一段时间后你的邮箱就99+啦

> [!Important]
>
> Github的`Discussions`、`Issues`和`Pull requests`都是用来解决bug/提出功能建议的，
> 所以你向一条Discussion回复Comment就会自动Subscribe，
> Subscribe后此条Discussion再被别人评论Github都会转发到你的邮箱里，无论是不是回复你的，由此来让你时刻关注进展

> [!NOTE]
>
> 经典案例：
> > EpicGames/Signup事件：
> > > 事故发生地：https://github.com/EpicGames/Signup/pull/24
> > > 当事人视频：https://www.bilibili.com/video/BV18Y4y1x7mc

> 绳网更新给我之前的Discussion整掉了

**Comment by Xmug** [comment]
好的

**Comment by bugmaibear** [comment]
感谢大佬

**Comment by KUNKUN4** [comment]
多谢提醒

**Comment by muxi0330** [comment]
这个委托我接了

**Comment by Callkit-P0p** [comment]
谢谢提醒
